travel.py

```diff
@@ -28,7 +29,7 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument('-c', '--cluster', help='Connection String i.e. localhost', default='db')
-parser.add_argument('-s', '--scheme', help='couchbase or couchbases', default='couchbase')
+parser.add_argument('-s', '--scheme', help='couchbase or couchbases', default='couchbases')
```

Comment:
this would default to how Capella needs the scheme to be defined to establish a connection.

Reply:
This presumably breaks the other invocations for local setup, so would need to be documented.

And if we go with couchbases as the default, that should arguably be applied to all the travel samples.

Probably a good idea to move to this, but needs a little care.

Reply:
reset back to couchbase